src/external/rawspeed/data/cameras.xml

```diff
@@ -6249,6 +6249,17 @@
 		<Crop x="0" y="0" width="4599" height="3064"/>
 		<Sensor black="0" white="4095"/>
 	</Camera>
+	<Camera make="SONY" model="DSLR-A380">
+		<ID make="Sony" model="DSLR-A380">Sony DSLR-A380</ID>
```

Comment:
You did check that the color matrix for this camera in src/external/adobe_coeff.c is the right one?
It is the same for 2 more cameras.